docs/en/source/gitGuideline.rst

```diff
@@ -0,0 +1,46 @@
+********************************
+Branch Management Guidelines 
+********************************
+
+.. figure:: figures/gitGuideline.jpg
+
+Community can have two main branches with an infinite lifetime:
+
+1. **Master branch**:This is a highly stable branch that is always production-ready and contains the last release version of source code in production.
+2. **Development branch**: Derived from the master branch, the development branch serves as a branch for integrating different features planned for an upcoming release. This branch may or may not be as stable as the master branch. It is where developers collaborate and merge feature branches.All of the changes should be merged back into master somehow and then tagged with a release number.
+
+
+Apart from those two primary branches, there are other branches in the workflow:
+
```

Comment:
I suggest to write Bug Branch and feature branch separately
feature branches are those branches which is for new feature addition to the Scorpio. Any issue present in Scorpio which is labeled as "enhancement"  or "documentation" will be fixed  under feature branch.
Bug branch is basically a branch which is for fixing any bug present Scorpio. Any issue present in Scorpio which is labeled as "Bug"  will be fixed  under feature branch.

Reply:
I think this is a good idea. So dev will become feature and a new bug branch. i would delay it a bit since we have bugs in dev at the moment. once we finished the bug reports on dev let's merge back to master and then start a new bug branch.